boot/bootutil/pkg.yml

```diff
@@ -39,5 +39,5 @@ pkg.deps:
     - "@apache-mynewt-core/sys/defs"
     - "@apache-mynewt-core/sys/flash_map"
 
-pkg.deps.BOOTUTIL_SIGN_EC256:
-    - "@apache-mynewt-core/crypto/tinycrypt"
+pkg.deps.BOOTUTIL_USE_TINYCRYPT:
```

Comment:
This does make a bit more sense, although currently, we do only need tinycrypt if we are using EC256 signatures.

Reply:
Ah, I suppose tinycrypt could be used for EC (224) too, correct?